From 934f2c03a918bc578e9ef849bdac291aae05ff0a Mon Sep 17 00:00:00 2001
From: jinghua <jinghua@marvell.com>
Date: Wed, 18 Nov 2015 08:11:39 -0800
Subject: [PATCH 0515/1240] fix: uart: armada3700: add BAUDRATE define in
 mvebu-common.h for armada3700

	in mvebu-common.h, BAUDRATE used to be defined when CONFIG_SYS_NS16550
	is defined.
	But for Armada3700, MVEBU_UART_ARLP uart driver is used, so
	BAUDRATE is missing.

	when compile Palladium image, there is another BAUDRATE in armadalp.h,
	which is specified for Palladium, so there is no compilation error.

Change-Id: I1d12112050a59f8d28ad74db597e769bc0f5997d
Signed-off-by: jinghua <jinghua@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/24994
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Omri Itach <omrii@marvell.com>
---
 include/configs/mvebu-common.h | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/include/configs/mvebu-common.h b/include/configs/mvebu-common.h
index 1441015..f842d60 100644
--- a/include/configs/mvebu-common.h
+++ b/include/configs/mvebu-common.h
@@ -236,11 +236,13 @@
 	#define CONFIG_SYS_NS16550_COM1		MVEBU_UART_BASE(0)
 	#define CONFIG_SYS_NS16550_CLK		200000000
 	#define CONFIG_CONS_INDEX		1
-	#define CONFIG_BAUDRATE			115200
 	#define CONFIG_SYS_LOADS_BAUD_CHANGE	/* allow baudrate changes */
 	/*#define CONFIG_SYS_DUART_CHAN		0*/
 #endif /* CONFIG_SYS_NS16550 */
 
+#if defined(CONFIG_SYS_NS16550) || defined(CONFIG_MVEBU_UART_ARLP)
+#define CONFIG_BAUDRATE		115200
+#endif
 
 /* GPIO */
 #ifdef MV_INCLUDE_GPIO
-- 
1.9.1

